### PR TITLE
Rename environments -> namespace-controller

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- rename chart environments to namespace-controller

**Related issue(s)**

### CODE REVIEW

File: resources/core/charts/namespace-controller/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Namespace controller
+
+## Overview
+
+Namespace controller is the name of the controller that injects limit ranges and resource quotas into user-created Namespaces. It also adds two default roles, `kyma-reader-role` and `kyma-admin-role`, which it then copies into user-created Namespaces.
+
+Read the [namespace-controller](https://kyma-project.io/docs/root/kyma#details-environments) document to learn more about Namespaces in Kyma.
```
Comment (klaudiagrz) [inline resources/core/charts/namespace-controller/README.md]: I guess the link also must be updated.

Comment (sjanota) [reply resources/core/charts/namespace-controller/README.md]: @tomekpapiernik will be updating docs in another PR. I cannot change this link until rest of docs is also renamed.

File: components/namespace-controller/README.md [hunk 2]
```diff
@@ -46,17 +46,17 @@ Make sure the image is built:
...
-If you have access to an external Docker registry, build your Docker image, push it to the registry and modify the [4-deployment.yaml](../../resources/core/charts/environments/templates/4-deployment.yaml) file by swapping the evironments image. Follow the [instructions](../../docs/kyma/docs/030-inst-local-installation-from-release.md) to run Kyma as usual.
+If you have access to an external Docker registry, build your Docker image, push it to the registry and modify the [4-deployment.yaml](../../resources/core/charts/namespace-controller/templates/4-deployment.yaml) file by swapping the evironments image. Follow the [instructions](../../docs/kyma/docs/030-inst-local-installation-from-release.md) to run Kyma as usual.
 
 #### Minikube built in Docker daemon
 
 In case you have no access to a Docker registry, use Minikube’s built in Docker daemon that keeps images for running containers:
 
-1. Modify the [4-deployment.yaml](../../resources/core/charts/environments/templates/4-deployment.yaml) file by swapping the evironments image.
+1. Modify the [4-deployment.yaml](../../resources/core/charts/namespace-controller/templates/4-deployment.yaml) file by swapping the evironments image.
```
Comment (Tomasz-Smelcerz-SAP) [inline components/namespace-controller/README.md]:
```suggestion
 1. Modify the [4-deployment.yaml](../../resources/core/charts/namespace-controller/templates/4-deployment.yaml) file by swapping the image version.
```

File: components/namespace-controller/README.md [hunk 3]
```diff
@@ -46,17 +46,17 @@ Make sure the image is built:
...
+Read the main [Kyma project README.md](../../README.md). By default, the system runs the namespace-controller image specified in the [4-deployment.yaml](../../resources/core/charts/namespace-controller/templates/4-deployment.yaml) file. You can provide your own image by following one of the procedures.
 
 #### Docker registry
 
-If you have access to an external Docker registry, build your Docker image, push it to the registry and modify the [4-deployment.yaml](../../resources/core/charts/environments/templates/4-deployment.yaml) file by swapping the evironments image. Follow the [instructions](../../docs/kyma/docs/030-inst-local-installation-from-release.md) to run Kyma as usual.
+If you have access to an external Docker registry, build your Docker image, push it to the registry and modify the [4-deployment.yaml](../../resources/core/charts/namespace-controller/templates/4-deployment.yaml) file by swapping the evironments image. Follow the [instructions](../../docs/kyma/docs/030-inst-local-installation-from-release.md) to run Kyma as usual.
```
Comment (Tomasz-Smelcerz-SAP) [inline components/namespace-controller/README.md]:
```suggestion
 If you have access to an external Docker registry, build your Docker image, push it to the registry and modify the [4-deployment.yaml](../../resources/core/charts/namespace-controller/templates/4-deployment.yaml) file by swapping the image version. Follow the [instructions](../../docs/kyma/docs/030-inst-local-installation-from-release.md) to run Kyma as usual.
```

**Pull request merged**